lib/sie/parser/line_parser.rb

```diff
@@ -5,13 +5,29 @@
 module Sie
   class Parser
     class LineParser
-      pattr_initialize :line
+      pattr_initialize :line, [:lenient]
```

Comment:
Not terribly important, but we generally do `[ :lenient ]`. I see we haven't been consistent in this library though.